psamm/commands/fastgapfill.py

```diff
@@ -105,12 +105,11 @@ def reaction_key(r):
         else:
             maximized_reaction = self._model.get_biomass_reaction()
             if maximized_reaction is None:
-                raise CommandError('The maximized reaction was not specified')
+                self.fail('The maximized reaction was not specified')
 
         if not self._mm.has_reaction(maximized_reaction):
-            raise CommandError(
-                'The biomass reaction is not a valid model'
-                ' reaction: {}'.format(maximized_reaction))
+            self.fail('The biomass reaction is not a valid model'
+                      ' reaction: {}'.format(maximized_reaction))
```

Comment:
I agree, we can keep the ones that check whether the reaction is a valid model reaction as `fail`.